File: LeetCode/1/1.py
#%%
def twoSum(nums: list[int], target: int) -> list[int]:
    data ={}
    for index, i in enumerate(nums):
        tmp = target - i
        if(tmp in data.keys()):
            print(data[tmp], index)
            return [data[tmp], index]
        data[i] = index

twoSum([3,3],6)
# %%

File: LeetCode/1/1.go
package main

// import "fmt"
func twoSum(nums []int, target int) []int {
    data := make(map[int]int)
	for index,i := range nums {
		tmp := target - i
		v, ok := data[tmp]
		if(ok){
			//fmt.Print(v,index)
			return []int{v, index}
		}
		data[i] = index
	}
	return []int{}
}

func main(){
	twoSum([]int{3,2,4}, 6)
}